zingolib/src/wallet/describe.rs

```diff
@@ -304,25 +304,94 @@ impl LightWallet {
     }
 }
 
-#[cfg(test)]
-mod tests {
+#[cfg(any(test, feature = "test-elevation"))]
+mod test {
+
+    use zcash_client_backend::PoolType;
+    use zcash_client_backend::ShieldedProtocol;
+    use zcash_primitives::consensus::NetworkConstants as _;
+
+    use crate::wallet::LightWallet;
+
+    // these functions are tested gold and a better pattern for address lookup.
```

Comment:
i dont know what you mean by "tested gold" but as I mentioned in the previous review these fns are incomplete and therefore are a worse pattern for address lookup. I am not asking to finish these functions as this is not the priority and they function for your tests now they are behind a test feature but there is no need to add this comment

Reply:
previous PR was merged without this change. could we implement this fix on this PR please?